core/che-core-tracing-core/src/main/java/org/eclipse/che/core/tracing/TracingInterceptor.java

```diff
@@ -47,45 +48,49 @@ public void init(Tracer tracer) {
   @Override
   public Object invoke(MethodInvocation invocation) throws Throwable {
     String spanName = getSpanName(invocation);
-    try (Scope scope =
+
+    Span span =
         tracer
             .buildSpan(spanName)
             .asChildOf(tracer.activeSpan())
             .withTag(Tags.SPAN_KIND.getKey(), Tags.SPAN_KIND_SERVER)
-            .startActive(true)) {
-
+            .start();
+    try (Scope scope = tracer.scopeManager().activate(span)) {
       Traced.TagsStack.push();
 
-      try {
-        return invocation.proceed();
-      } finally {
-        for (Map.Entry<String, Supplier<?>> e : Traced.TagsStack.pop().entrySet()) {
-          Object val;
-          try {
-            val = e.getValue().get();
-          } catch (Exception ex) {
-            if (LOG.isDebugEnabled()) {
-              // we want to know the exception in case the tag extraction failed.
-              // Slf4j doesn't seem to provide a method overload that could both provide formatting
-              // arguments and the cause.
-              LOG.debug(
-                  "Could not get the value for a tag called {} when tracing {}.",
-                  e.getKey(),
-                  spanName,
-                  ex);
-            }
-            continue;
+      return invocation.proceed();
+    } catch (Exception e) {
+      LOG.error(e.getMessage(), e);
+      span.log(e.getMessage());
+      throw e;
+    } finally {
+      for (Map.Entry<String, Supplier<?>> e : Traced.TagsStack.pop().entrySet()) {
+        Object val;
+        try {
+          val = e.getValue().get();
+        } catch (Exception ex) {
+          if (LOG.isDebugEnabled()) {
+            // we want to know the exception in case the tag extraction failed.
```

Comment:
I know this was here before and that mostly possibly I'm the author, but this comment should IMHO be removed. It refers to the code that's not actually here :)